### PR TITLE
[TDD] Kata - RPN Calculator

### PR DESCRIPTION
**Pull request by sahaparamjit**

### CODE REVIEW

**Review by sahaparamjit — COMMENTED** [review]
review the PR

File: src/test/java/com/ordestiny/tdd/kata/RPNCalculatorTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    private static RPNCalculator rpnCalculator;
+
+    @BeforeAll
+    public static void init() {
+        rpnCalculator = new RPNCalculator();
+    }
+
+    @ParameterizedTest
+    @MethodSource("testData")
+    void calculate(String postExpr, int expected) {
```
Comment (fengyuanyang) [inline src/test/java/com/ordestiny/tdd/kata/RPNCalculatorTest.java:22]: Hi , 
it seems you didn't pass the coverage check.
[ant:jacocoReport] Rule violated for class com.ordestiny.tdd.kata.RPNCalculator: lines covered ratio is 0.85, but expected minimum is 0.95

> Task :jacocoTestCoverageVerification FAILED
FAILURE: Build failed with an exception.
2 actionable tasks: 1 executed, 1 up-to-date

* What went wrong:
Execution failed for task ':jacocoTestCoverageVerification'.
> Rule violated for class com.ordestiny.tdd.kata.RPNCalculator: lines covered ratio is 0.85, but expected minimum is 0.95

you could use gradle tasks
such as 
gradle task jacocoTestReport to see, which lines are not covered

**Review by fengyuanyang — COMMENTED** [review]
please check my comments

File: src/test/java/com/ordestiny/tdd/kata/RPNCalculatorTest.java (hunk 1 above)
Comment (sahaparamjit) [reply src/test/java/com/ordestiny/tdd/kata/RPNCalculatorTest.java:25]: @fengyuanyang addressed

**Comment by fengyuanyang** [comment]
@sahaparamjit 
Thanks, LGTM

**Pull request merged**